extensions/console/02_console.js

```diff
@@ -195,8 +195,8 @@
       const needed = (columnWidths[i] - len) / 2;
       // round(needed) + ceil(needed) will always add up to the amount
       // of spaces we need while also left justifying the output.
-      out += `${StringPrototypeRepeat(" ", needed)}${cell}${
-        StringPrototypeRepeat(" ", MathCeil(needed))
+      out += `${cell}${StringPrototypeRepeat(" ", MathCeil(needed))}${
+        StringPrototypeRepeat(" ", needed)
       }`;
```

Comment:
Per [Simon's observation](https://github.com/denoland/deno/pull/11295/files#r664785314) (2/2)

```suggestion
      out += `${cell}${StringPrototypeRepeat(" ", needed)}`;
```